about.html

```diff
@@ -0,0 +1,245 @@
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+      <header class="container-fluid desktop-head">
+        <div class="container">
+          <div class="row">
+            <div class="col upper-head">
+              <h3 class="header-text-a">Welcome! Blessed Adegu Initiative</h3>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col">
+              <h1 class="header-text-b">
+                A WHOLE ROUNDED COUNSELLING SERVICES 2022
+              </h1>
+            </div>
+          </div>
+          <div class="row header-tect-c-div">
+            <div class="col">
+              <p class="header-text-c">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
+                mollitia, molestiae quas vel sint commodi repudiandae
+                consequuntur voluptatum laboruLorem ipsum dolor sit amet
+                consectetur adipisicing elit. Maxime mollitia, molestiae quas
+                vel sint commodi repudiandae consequuntur voluptatum laboru
+              </p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col">
+              <p class="header-text-e">
+                Please contact us per email for further questions about Blessed
+                Adegu <br />
+                <strong>info@blessedadegu.com</strong>
+              </p>
+            </div>
+          </div>
+        </div>
+      </header>
+    </section>
+
+    <section class="middle-section">
+      <div class="middle-section-head">
+        <h2>Counselling for <br />the Greate Minds!</h2>
+        <hr />
+      </div>
+
+      <div class="top-program-1">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
+          mollitiaLorem ipsum dolor sit amet consectetur adipisicing elit.
+          Maxime mollitia Lorem ipsum dolor sit amet consectetur adipisicing
+          elit. Maxime mollitia
+        </p>
+      </div>
+
+      <div class="top-program-2">
+        <img src="./images/partner/images.png" alt="" />
+      </div>
+    </section>
+    <section class="middle-section">
+      <div class="middle-section-head">
+        <h2>See the past <br />blessed Adegu events</h2>
+        <hr />
+      </div>
+      <div class="col top-program-1">
+        <p>Take a look at the previous sessions that we've held</p>
+      </div>
+      <div class="row">
+        <div class="col-sm-6 col-md-2 top-program-2b">
+          <h4>2020</h4>
+          <p>Harmony SDA Church sessions</p>
+        </div>
+        <div class="col-sm-6 col-md-2 top-program-2b">
+          <h4>2021</h4>
+          <p>Namunyiri SDA church sessions</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="patners">
+      <div class="row outer-bottom-partner-head">
+        <div class="col-md-6 bottom-partner-head">
+          <h2>Our Sponsors</h2>
+          <hr />
+        </div>
+      </div>
```

Comment:
- [ ] Kindly horizontally center logos in `Our Sponsors section`. Now they are aligned to the left. 
<img width="644" alt="Screenshot 2022-08-04 at 10 00 59" src="https://user-images.githubusercontent.com/77038610/182774099-c2bf9ba8-e0ec-46a8-9fe9-6895340b6fe5.png">
 

- [ ] Kindly use colors as provided in Design, the `Sponsors section` is black, but should be #271A31 color
<img width="379" alt="Screenshot 2022-08-04 at 10 02 01" src="https://user-images.githubusercontent.com/77038610/182774238-b99a305e-4651-477a-bbb2-b76c4a0ed89b.png">

- [ ] In mobile size,` Sponsors section` is breaking, kindly fix it

<img width="377" alt="Screenshot 2022-08-04 at 10 03 42" src="https://user-images.githubusercontent.com/77038610/182774485-d8237236-294b-409b-9177-9f3581948612.png">